apps/epl/src/epl_tracer.erl

```diff
@@ -239,6 +270,10 @@ handle_info(timeout,
                                     Proplist,
                                     EtsTables),
 
+            {value, {timeline, Timelines}, PropsListWithoutFaultyTimelines } = lists:keytake(timeline, 1, Proplist1),
```

Comment:
On the "tracer side" I store pids as pids. But when I transfered them to epl node they changed (because of CreationBit (or byte can't recall right now)). Thus I removed them, translated them to lists (because they do consist only of 3 number) so they'll be consistent on both sides.